lstchain/reco/utils.py

```diff
@@ -829,3 +830,24 @@ def apply_src_r_cut(events, src_r_min, src_r_max):
     ]
 
     return events
+
+def get_events_in_GTI(events, CatB_cal_table):
```

Comment:
Is this function used somewhere in the current PR?

Reply:
No, this is of help for the DL2 analysis

Reply:
since the concept of GTI is more general than the monitoring data from Cat B calibration, I would name this function differently, something like `get_events_in_CatB_GTI`. But I think this should be done in another PR

Reply:
I think we can keep the name and expand the function to include more information once we have more things that define the final GTIs?

Reply:
agreed